great_expectations/optional_imports.py

```diff
@@ -10,10 +10,31 @@
 from __future__ import annotations
 
 import warnings
+from typing import Any
 
 from packaging.version import Version
 
 
+class NotImported:
```

Comment:
This was moved from `util.py` since we are trying to stop using this generically named file.